Test/Framework/Visual/VertexBufferTest.cs

```diff
@@ -121,7 +121,36 @@ public void GetPosition(bool dynamic)
                 Assert.AreEqual(savedData[2].Position, readData[2]);
                 Assert.AreEqual(savedData[3].Position, readData[3]);
             };
-            Game.RunOneFrame();
+            Game.Run();
```

Comment:
You mean you can step into, say, `Texture2DTest.ShouldSetAndGetData`, and see it actually go into the `Game.DrawWith` event handler?

I'm using ReSharper. Should I be using something else?